packages/fetchai/agents/tac_participant/aea-config.yaml

```diff
@@ -33,3 +33,6 @@ default_routing:
   fetchai/ledger_api:0.10.0: fetchai/ledger:0.13.0
   fetchai/oef_search:0.13.0: fetchai/soef:0.16.0
 connection_private_key_paths: {}
+decision_maker_handler:
+  dotted_path: aea.decision_maker.gop:DecisionMakerHandler
```

Comment:
This is how `aea` specific DMs can be modified (default points to `aea.decision_maker.default:DecisionMakerHandler`)